core/toolbox/toolbox.js

```diff
@@ -13,56 +13,54 @@
 goog.module('Blockly.Toolbox');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly');
-goog.require('Blockly.BlockSvg');
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.CollapsibleToolboxCategory');
-goog.require('Blockly.ComponentManager');
-goog.require('Blockly.Css');
-goog.require('Blockly.DeleteArea');
-goog.require('Blockly.Events');
+const Blockly = goog.require('Blockly');
+const BlocklyOptions = goog.requireType('Blockly.BlocklyOptions');
```

Comment:
Needs `/* eslint-disable-next-line no-unused-vars */` (here and ~9 more places below).